azure-pipelines.yml

```diff
@@ -118,37 +122,4 @@ jobs:
         rerunMaxAttempts: 2
       env:
         AzureWebJobsStorage: $(AzureWebJobsStorage)
-        APPINSIGHTS_INSTRUMENTATIONKEY: $(APPINSIGHTS_INSTRUMENTATIONKEY) 
-    
-  - job: PublishPipelineArtifact
```

Comment:
@amdeel, I think this is safe to remove, as we only need publishing on our releases, not our CI, but I'd like your input.

Reply:
This is just to publish the nuget package of this build in the artifacts dropdown, in case someone wants a quick nuget package for testing. It's an insignificant performance cost to have this job. Admittedly though, I've never used this. And if I'm testing my own changes I'll just make my own nuget package. I assume if we are testing nuget packages as a team we will likely upload our nuget package to myget. So I'm not opposed to removing this.

Reply:
Since it has the "dependsOn" clause, it runs only after all of the tests have passed, meaning even though the task itself only takes a few minutes, it adds time until the CI is marked as green.